### ISSUE TITLE
Исключить из парсинга блок "рекомендуем к просмотру"

### ISSUE THREAD

**Issue by olegon85** [Duff89/parser_avito#72]
Авито вставляет блок "рекомендуем к просмотру" Возможно ли исключить из парсинга этот блок? Скорее всего исключив этот блок из парсинга, получится немного увеличить скорость и избавится от посторонних объявлений. не понадобится указывать город и загружать доп параметры

**Comment by whitebubblegum** [comment]
Согласен, было бы славно как-нибудь игнорировать этот блок, если это возможно. Почти все уведомления состоят из этого, очень сложно становится отслеживать нужные объявления среди них. Интересно услышать мнение автора)

**Comment by Duff89** [comment]
Запланировано

**Comment by Duff89** [comment]
реализовано
